resources/windows/wix/properties.wxs.erb

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+
+    <ComponentGroup Id="Include_Properties" />
+
+    <!-- Add/Remove Programs icon and help link -->
+    <Icon Id="$(var.OurCompanyNameWord).ico" SourceFile="conf\windows\stage\misc\puppetlabs.ico"/>
```

Comment:
`$(var.OurCompanyNameWord)` needs to be moved to an erb variable, and the location of `puppetlabs.ico` needs to be defined